content/pt-br/_index.html

```diff
@@ -43,12 +43,12 @@ <h2>Os desafios da migração de mais de 150 micro serviços para o Kubernetes</
         <button id="desktopShowVideoButton" onclick="kub.showVideo()">Assista Video</button>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccnceu22" id="desktopKCButton">KubeCon na Europa de 16 a 20 de maio de 2022</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/" id="desktopKCButton">KubeCon + CloudNativeCon na Europa de 19 a 22 de março de 2024</a>
         <br>
         <br>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccncna22" button id="desktopKCButton">KubeCon na América do Norte de 24 a 28 de outubro de 2022</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america-2024/" button id="desktopKCButton">KubeCon + CloudNativeCon na América do Norte de 24 a 28 de novembro de 2024</a>
```

Comment:
Dates seem incorrect in this line

```suggestion
        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america-2024/" button id="desktopKCButton">KubeCon + CloudNativeCon na América do Norte de 12 a 15 de novembro de 2024</a>
```

Reply:
Done, thanks